.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: Bug report
+description: File a bug report to help us improve
+title: "[BUG]"
+labels: ["bug", "🔍 needs-triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Contributing directly to this project is always encouraged. More details can be found in the [Contributing Guide](https://github.com/Azure/template-analyzer/blob/development/CONTRIBUTING.md).
```

Comment:
I think this will work...
``` suggestion
        Thanks for taking the time to fill out this bug report! Contributing directly to this project is always encouraged. More details can be found in the [Contributing Guide](../../CONTRIBUTING.md).
```